mouhamadouBarry/quiz/q1.js

```diff
@@ -0,0 +1,67 @@
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+/*
+INPUT: 
+    - An array of positve numbers
+    - A positive k int
+OUTPUT:
+    - Find the maximum sum of any contigous subarray of size k
+
+Overview:
+    - This problem look like a sliding windows problem
+        -> startPtr 
+        -> endPtr
+        -> max
+        -> sum
+
+    - I will use a while loop to iteravte through out the array
+        ->As the loop goes beyond k, I will subtract the starting pointer
+Edge Cases:
+    - What if the k is larger then the array?
+    - What if the array is empty?
+    - I'm I allow to modify the original array?
+    - will 0 be considered a positive number? if so, What if k is 0?

```

Comment:
great use of high level overview, thank you for detailing questions you will ask the interviewer in edge cases.